.github/workflows/macos.yml

```diff
@@ -45,6 +45,7 @@ jobs:
             --disable-pacct
             --disable-smtp
             --disable-mqtt
+            --disable-grpc
```

Comment:
This package should have the C++ plugin for protoc: https://formulae.brew.sh/formula/grpc

Reply:
I think there is a linker bug on macos :/

```
2023-07-03T12:15:04.6237110Z 0  0x10c3e6f7a  __assert_rtn + 139
2023-07-03T12:15:04.6242110Z 1  0x10c285868  ld::tool::InputFiles::findDylib(char const*, ld::dylib::File const*, bool) + 1112
2023-07-03T12:15:04.6246580Z 2  0x10c36c242  generic::dylib::File::processIndirectLibraries(ld::dylib::File::DylibHandler*, bool) + 82
2023-07-03T12:15:04.6250380Z 3  0x10c28740b  ld::tool::InputFiles::createIndirectDylibs() + 571
2023-07-03T12:15:04.6255350Z 4  0x10c2883d3  ld::tool::InputFiles::forEachInitialAtom(ld::File::AtomHandler&, ld::Internal&) + 1059
2023-07-03T12:15:04.6259780Z 5  0x10c29ed63  ld::tool::Resolver::resolve() + 147
2023-07-03T12:15:04.6264260Z 6  0x10c1f8917  main + 887
2023-07-03T12:15:04.6267660Z A linker snapshot was created at:
2023-07-03T12:15:04.6269610Z 	/tmp/libotel.dylib-2023-07-03-121504.ld-snapshot
2023-07-03T12:15:04.6270560Z ld: Assertion failed: (counter() < 0xffff), function nextCounter, file ld.hpp, line 466.
2023-07-03T12:15:04.6438050Z collect2: error: ld returned 1 exit status
2023-07-03T12:15:04.6480550Z make[2]: *** [modules/grpc/otel/libotel.la] Error 1
2023-07-03T12:15:04.6481920Z make[2]: Target `all-am' not remade because of errors.
2023-07-03T12:15:04.6496820Z make[1]: *** [all-recursive] Error 1
2023-07-03T12:15:04.6504540Z make: *** [all] Error 2
```